components/chunkprogressbar/overview.md

```diff
@@ -13,11 +13,11 @@ position: 0
 The <a href = "https://www.telerik.com/blazor-ui/chunkprogressbar" target = "_blank">Blazor ChunkProgressBar component</a> tracks the execution of operations and displays what portion of it is completed in predefined number of sections (chunks). For very long tasks, you can also make it [indeterminate]({%slug chunkprogressbar-indeterminate-state%}) while waiting for an update.
```

Comment:
The structure of this article does not follow our Overview page template.
* The Parameters section should come later.
* The headings are too long.
* Some of the links in the Chunk Count section make no sense, as they lead to content above this section.
* The Matching and Non-Matching sections should go to a separate article.

Reply:
* Moved the parameters back to the bottom.
* Reduced the heading length
* I am not sure I understand which links make no sense tho. 
* The reason why I think we should keep them in this article is that they are the bread and butter of how to set up the component. This is the most important concept of the component and I believe it must still be in the Overview.